README.md

```diff
@@ -2,6 +2,29 @@
 
 Initialize bitrise config, step template or plugin template
 
+## How to build this project 
+Project is written in [Go](https://golang.org/) language and 
+uses [godep](https://github.com/tools/godep) as dependency management tool.
+
+You can build this project using sequence of `go` commands or refer to [bitrise.yml](./bitrise.yml) file,
+which contains workflows for this project.
+
+You can run `bitrise` workflows on your local machine using [bitrise CLI](https://www.bitrise.io/cli).
+
+Before you start, make sure 
+- `$HOME/go/bin` is added to `$PATH`
+- `Ruby >= 2.2.2` version is installed (required for integration tests)
+
+**How to build the project using Go commands**
+- `go build` command builds the project and generates `bitrise-init` binary at `$HOME/go/bin/bitrise-init`.
+- `go test ./...` command runs unit tests in every project folder/subfolder.
+- `go test -v ./_tests/integration/...` command runs integration tests. This command requires `INTEGRATION_TEST_BINARY_PATH=$HOME/go/bin/bitrise-init` environment variable.
+
+**How to build the project using bitrise workflows**
```

Comment:
Please change the order of **How to build the project using Go commands** and **How to build the project using bitrise workflows**, since building and testing this project using the bitrise-cli is the preferred way